apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/TransactionImpl.java

```diff
@@ -267,6 +295,7 @@ BuilderImpl index(short index) {
     private volatile DbKey dbKey;
     private volatile byte[] bytes = null;
 
+    public TransactionImpl() {} // for weld
```

Comment:
I think, we dont need TransactionImpl using under CDI.

Reply:
Removed.